corehq/apps/analytics/management/commands/initialize_kissmetrics_plan_editions.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management import BaseCommand
+from corehq.apps.accounting.signals import subscription_upgrade_or_downgrade
+from corehq.apps.domain.models import Domain
+
+
+class Command(BaseCommand):
+    help = ("Send KISSmetrics data on subscriptions for all domains")
+
+    def handle(self, *args, **options):
+        for domain in Domain.get_all():
+            subscription_upgrade_or_downgrade.send_robust(domain, domain=domain)
```

Comment:
Should the first argument here be `Domain` (the class)? If not, seems like the second argument is redundant? These docs suggest you use the class as the sender arg, but I didn't read far enough to find out why: https://docs.djangoproject.com/en/1.6/topics/signals/#django.dispatch.Signal.send_robust